deepspeed/module_inject/auto_tp.py

```diff
@@ -309,6 +309,10 @@ def tp_parser(model):
                     gem_list = gem_list + [layer]
```

Comment:
@Yejing-Lai - could you resolve merge conflicts on this PR then we can get it merged?

Reply:
Conflict resovled~

Reply:
Thanks!